addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -92,11 +92,15 @@ def _get_sale_order_line_multiline_description_sale(self):
 
     def _get_display_price(self):
         if self.event_booth_pending_ids and self.event_id:
-            company = self.event_id.company_id or self.env.company
-            currency = company.currency_id
-            total_price = sum([booth.price for booth in self.event_booth_pending_ids])
-            return currency._convert(
-                total_price, self.order_id.currency_id,
-                self.order_id.company_id or self.env.company.id,
-                self.order_id.date_order or fields.Date.today())
+            return sum([super(SaleOrderLine,
+                              self.with_context(
+                                  pricelist=self.order_id.pricelist_id.id,
+                                  record_being_sold=booth._origin.booth_category_id.id,
+                                  quantity=1)  # Assume there are no duplicate booth -> quantity = 1
+                              )._get_display_price() for booth in self.event_booth_pending_ids])
```

Comment:
A few ideas here:
* It may be worthwhile to create and reuse a `pricelist_id = self.order_id.pricelist_id.id` as we're in 'ensure_one'
* I find it risky to silently override `quantity` here in case it was set, whether it should or not. Also, is there nothing to keep from `self._context` at this point?
* You can pass a generator to `sum` and avoid making a list first with the data.

Reply:
1. Yes indeed we can extract pricelist_id out of the loop
2. Without the code that count the number of type of booth in the order line, I think quantity must be assumed to be 1 for the computation of the unit price. We have removed that code because it is not supported in the rest of the code and that such code would be misleading then. About what to keep in the context: as far as I know using the method like that, we keep everything beside what is overridden by the 3 arguments.
3. Yes indeed, stupid mistake, thanks